### ISSUE TITLE
Redundant logging when connection failed.

### ISSUE THREAD

**Issue by madongfly** [grpc/grpc-java#1102]
And SEVERE is too heavy for it.

It should be fixed in Netty, but create this issue for gRpc users tracking it.

```
Oct 05, 2015 2:00:16 PM io.netty.handler.codec.http2.Http2ConnectionHandler processGoAwayWriteResult
SEVERE: Sending GOAWAY failed: lastStreamId '3', errorCode '0', debugData ''. Forcing shutdown of the connection.
javax.net.ssl.SSLException: SSLEngine closed already
```

**Comment by nmittler** [comment]
@ejona86 what exactly do we want to change here? Has anyone raised the Netty issue?

**Comment by ejona86** [comment]
I think this has been fixed already in netty/netty#4938.
